tools/scalar-schema/README.md

```diff
@@ -0,0 +1,80 @@
+# Schema Tool for Scalar DB
+This tool makes schemas on Cassandra or Cosmos DB for [Scalar DB](https://github.com/scalar-labs/scalardb).
+  - This tool creates databases(collections) and tables(containers), also inserts metadata which is required by Scalar DB for Cosmos DB
+
+# Usage
+
+## Build & Run
+### Build a standalone jar
+```console
+$ cd /path/to/scalardb/tools/scalar-schema
+$ lein uberjar
+```
+
+### Create tables
+```console
+# For Cosmos DB
+$ java -jar target/scalar-schema.jar --cosmos -h <YOUR_ACCOUNT_URI> -p <YOUR_ACCOUNT_PASSWORD> -f schema.json [-r BASE_RESOURCE_UNIT]
+```
+  - `-r BASE_RESOURCE_UNIT` is an option. You can specify the RU of each database. The maximum RU in tables in the database will be set. If you don't specify RU of tables, the database RU will be set with this option. When you use transaction function, the RU of the coordinator table of Scalar DB is specified by this option. By default, it's 400.
+
+```console
+# For Cassandra
+$ java -jar target/scalar-schema.jar --cassandra -h <CASSANDRA_IP> -u <CASSNDRA_USER> -p <CASSANDRA_PASSWORD> -f schema.json [--network-strategy <NETWORK_STRATEGY> --replication-factor <REPLICATION_FACTOR>]
```

Comment:
Just a question. `--network-strategy` needs to start with double dashes?  Can it also be `-ns` or `-rf` if they are configured so?

Reply:
`clojure/tools.cli` doesn't support multi-characters with a single dash. I added short options for them.

```
% java -jar target/scalar-schema.jar --help
      --cassandra                                   Operate for Cassandra
      --cosmos                                      Operate for Cosmos DB
  -h, --host DB_HOST                                Address of Cassandra like IP or URI address of your Cosmos DB account
  -u, --user USERNAME                               Username of the database. This option is ignored when Cosmos DB.
  -p, --password ACCOUNT_PASSWORD                   Password of Cassandra or Cosmos DB account
  -f, --schema-file SCHEMA_JSON                     Schema file
  -r, --ru RESOURCE_UNIT                       400  Base RU for each table on Cosmos DB. The RU of the coordinator for Scalar DB transaction is specified by this option. This option is ignored when Cassandra.
  -R, --replication-factor REPLICATION_FACTOR  1    The number of replicas. This options is ignored when Cosmos DB.
  -n, --network-strategy NETWORK_STRATEGY           The network topology strategy. SimpleStrategy or NetworkTopologyStrategy. This options is ignored when Cosmos DB.
  -D, --delete-all                                  All database will be deleted, if this is enabled.
      --help
```